ui/components/app/modals/transaction-already-confirmed/transaction-already-confirmed.component.js

```diff
@@ -0,0 +1,59 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalBody,
+  Text,
+  ModalFooter,
+} from '../../../component-library';
+import {
+  AlignItems,
+  FlexDirection,
+} from '../../../../helpers/constants/design-system';
+import { I18nContext } from '../../../../contexts/i18n';
+
+export default function TransactionAlreadyConfirmed({
```

Comment:
Could you add a comment explaining when we show this modal? Something like "This modal is displayed when user tries to submit a cancel transaction when the original transaction has been already confirmed".